DangoPlop/Assets/Scripts/Ball_Behavioiur.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Ball_Behavioiur : MonoBehaviour {
+    public GameObject player;
+    public GameObject ball;
+    public Vector2 ballPos;
+    public Rigidbody2D rb;
```

Comment:
Can we make these private. Things that are public but do not have to be, should just be private.